Drop-In/src/main/java/com/braintreepayments/api/dropin/BaseActivity.java

```diff
@@ -25,9 +29,23 @@
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
+        if (savedInstanceState != null) {
+            try {
+                mConfiguration = Configuration.fromJson(savedInstanceState.getString(EXTRA_CONFIGURATION_DATA));
+            } catch (JSONException e) {}
```

Comment:
rename `e` to `ignored` so Android studio won't complain about an empty catch block.